tests/test_ui.py

```diff
@@ -321,10 +321,34 @@ def leases4_250(kea: KeaClient) -> None:
         )
 
 
+@pytest.fixture(scope="function")
+def netbox_user_permissions() -> list[dict]:
+    return [{"actions": [], "object_types": []}]
+
+
 @pytest.fixture(scope="function", autouse=True)
 def netbox_login(
-    page: Page, netbox_url: str, netbox_username: str, netbox_password: str
+    page: Page,
+    netbox_url: str,
+    netbox_username: str,
+    netbox_password: str,
+    netbox_user_permissions: list[dict],
```

Comment:
nit: could you change all `list[dict]` to `list[dict[str, Any]]` or `list[dict[str, list[Any]]]`

Reply:
Sure 👍